esmf_regrid/experimental/unstructured_scheme.py

```diff
@@ -174,17 +212,75 @@ def _regrid_unstructured_to_rectilinear__perform(src_cube, regrid_info, mdtol):
 
 
 def regrid_unstructured_to_rectilinear(src_cube, grid_cube, mdtol=0):
-    """TODO: write docstring."""
+    """
+    Regrid unstructured cube onto rectilinear grid.
+
+    Return a new cube with data values calculated using the area weighted
+    mean of data values from unstructured cube src_cube regridded onto the
+    horizontal grid of grid_cube.
```

Comment:
Could mention here the dimensions that the output cube takes, wrt to `src_cube` and `grid_cube`